regexes/device/mobiles.yml

```
@@ -13025,12 +13039,45 @@ BDF:
 
 # Highscreen
 Highscreen:
-  regex: '(Power Ice Max|Zera[ _][UFS])[;)/ ]'
+  regex: '(Power Ice Max|Power Rage Evo|Zera[ _][UFS]|PowerFivePro|PowerFour)[;)/ ]'
   device: 'smartphone'
-  model: '$1'
+  models:
+    - regex: '(Power Ice Max|Power Rage Evo|Zera[ _][UFS])[;)/ ]'
+      model: '$1'
+    - regex: 'PowerFivePro[;)/ ]'
+      model: 'Power Five Pro'
+    - regex: 'PowerFour[;)/ ]'
+      model: 'Power Four'
 
 # CVTE
 CVTE:
   regex: 'CVTE[_ ](MSD338_(:?1G|512M))[;)/ ]'
   device: 'tv'
-  model: '$1'
\ No newline at end of file
+  model: '$1'
+
+# Globex
+Globex:
+  regex: '(GU8012C)[;)/ ]'
```

```suggestion
  regex: 'GU8012C[;)/ ]'
```

there is currently no need for those brakets